packages/main/src/Link.js

```diff
@@ -115,6 +115,18 @@ const metadata = {
 			defaultValue: "",
 		},
 
+		/**
+		 * Use aria-expanded to indicate the state of a collapsible element.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 * @since 1.1.0
+		 */
+		 ariaExpanded: {
```

Comment:
IMO: We should not encourage setting native aria attributes on custom element level. Maybe the the property should be just expanded and should be public. This would lead with a component with no role and aria-attribute set on top, which is not correct.

"Determines if the component has opened a popup." @ilhan007  what do you think?

Reply:
Hello @elenastoyanovaa, @ilhan007,
I didn't understand, should we add <code>role="link"</code> to the anchor tag with this change?